### ISSUE TITLE
Can't crawl localhost

### ISSUE THREAD

**Issue by Olyno** [jaeles-project/gospider#44]
Hi 👋🏻 

I tried to crawl ``http://localhost:3000`` but looks like i can't: ``[0000] ERROR Failed to parse domain``

How could i fix that please?

Using the latest commit.

**Comment by ocervell** [comment]
+1

**Comment by JodisKripe** [comment]
Try http://localhost.localdomain

If it doesn't work, You can add that to your /etc/hosts

Something Like this

`[jodis@fedora ~]$ cat /etc/hosts
# Loopback entries; do not change.
# For historical reasons, localhost precedes localhost.localdomain:
127.0.0.1   localhost localhost.localdomain localhost4 localhost4.localdomain4
::1         localhost localhost.localdomain localhost6 localhost6.localdomain6
`

**Comment by ocervell** [comment]
imo we shouldn't need to have a to specify an aliased domain for this, we should be able to scan `localhost` directly ...

**Comment by JodisKripe** [comment]
> imo we shouldn't need to have a to specify an aliased domain for this, we should be able to scan `localhost` directly ...

Yeah agreed, but I think the issue faced was resolved as soon as there was a 'period'(.) in the domain. 
This could be fixed by the Team, but this is like, the temporary fix. I'm not 100% sure that's the reason for the issue, but this seemed to work for me then.
